spec/index.bs

```diff
@@ -303,16 +302,22 @@ interface GPUTextureView : GPUObjectBase {
 
 <script type=idl>
 dictionary GPUTextureViewDescriptor : GPUObjectDescriptorBase {
-    required GPUTextureFormat format;
-    required GPUTextureViewDimension dimension;
-    required GPUTextureAspect aspect;
+    GPUTextureFormat format;
+    GPUTextureViewDimension dimension;
+    GPUTextureAspect aspect = "all";
     u32 baseMipLevel = 0;
     u32 mipLevelCount = 1;
```

Comment:
Should "default texture view" cover all the mipmap levels and the array layers instead of the first ones?

Reply:
it absolutely should (great point!), if we are going this way
we could make the `mipLevelCount=0` to have this special semantics here

Reply:
I agree. Would you open another PR for this?

Reply:
Done in [#407 ](https://github.com/gpuweb/gpuweb/pull/407). PTAL, thanks!